core/src/test/java/io/fabric8/maven/core/util/ThorntailUtilTest.java

```diff
@@ -0,0 +1,19 @@
+package io.fabric8.maven.core.util;
```

Comment:
I think license plugin is crying because of this file

Reply:
But what's happening?

Reply:
ok now :)